tools/enso4igv/IGV.md

```diff
@@ -17,9 +17,9 @@ major operating systems.
 
 ## Installation
 
-Visit [GraalVM's IGV page](https://www.graalvm.org/jdk22/tools/igv/) to read and
+Visit [GraalVM's IGV page](https://www.graalvm.org/22.1/tools/igv/) to read and
 download _IGV_. Or follow
-[this link](https://lafo.ssw.uni-linz.ac.at/pub/idealgraphvisualizer/idealgraphvisualizer-0.31-cb98bbf5fef-all.zip)
+[this link](https://lafo.ssw.uni-linz.ac.at/pub/idealgraphvisualizer/idealgraphvisualizer-1.20-ea624c6066a.zip)
```

Comment:
@Akirathan, @4e6, @hubertp please make sure you upgrade your IGV.

Reply:
@Akirathan, what's the best way to regenerate **all** `.enso-sources*` files? Is there a way to compile everything in our `sbt` based system? Right now I have problems with wrappers like `akka-wrapper` - Not even `clean` and `compile` works for them. At the end I did:
```bash
enso$ rm -rf *
enso$ git checkout -f
enso$ sbt
sbt:enso> buildEngineDistribution
sbt:enso> testOnly Nothing
sbt:enso> bench^C
```
However that's very tedious. Do you think there is a way to invoke all Java compilations only? Moreover just generate the `.enso-sources*` files, but don't perform any compilation (that might be done with a simple property check in `FrgaalJavaCompiler.scala`)? E.g. a command to do all Java compilations (main, test, bench, etc.) for all aggregated projects is what I am looking for.

Reply:
`akka-wrapper/compile` does not do anything on purpose. The "correct" command to compile this project is `akka-wrapper/exportedModule`. The thing is that it is not possible to change the default behavior of the `compile` command without breaking everything else in sbt. Therefore, the new command.

But I understand your frustration in the context of this task. Generating `.enso-sources*` files is currently done in `FrgaalJavaCompiler` as part of the `compile` command, but it seems that it is, in fact, independent of the compilation and therefore, can be moved to a completely independent task. If you are interested in this pursuit, please, create an issue for that and assign it to me.

Reply:
I can type `sbt:enso> exportedModule` - if that is the right command to force compilation even of `akka-wrapper`.

Reply:
I am not sure that will work. `buildEngineDistributionNoIndex` works for sure - it invokes `exportedModule` for all the dependencies that are build into the `built-distribution/.../component` directory, including all the wrappers.